internal/services/registry/storage.go

```diff
@@ -138,6 +140,42 @@ func (p *proc) Handle(ctx context.Context, data *ceData) error {
 				}
 			}
 		}
+	case mtr.R.BurnRequestVehicleNFT != nil:
```

Comment:
I would imitate the claim handler and have this basically be a no-op.

Put the burn "handler" in the contract event processor so that you can handle the raw event if someone burns directly on-chain.